skiko/build.gradle.kts

```diff
@@ -348,10 +351,12 @@ val wasmCompile = project.tasks.register<Exec>("wasmCompile") {
         dependsOn(skiaWasmDir)
         val skiaDir = skiaWasmDir.get().absolutePath
         val outDir = "$buildDir/wasm"
-        val srcs = listOf(
+        val srcs = mutableListOf(
             "$projectDir/src/jsMain/cpp",
             "$projectDir/src/commonMain/cpp"
-        ).findAllFiles(".cc").toTypedArray()
+        ).apply {
+            if (skiko.includeTestHelpers) add("$projectDir/src/commonTest/cpp/TestHelpers.cc")
```

Comment:
Why conditionally? Cannot we have two .wasm artefacts?

Reply:
Yes, we can have skiko-test-helpers.wasm to include it in js tests. We also can have skiko-test-helpers.a and include it in native tests.
I didn't manage to build a separate lib for jvm tests though (couldn't configure CppLibrary to make it build two separate libs (skiko and skiko-test-helpers). Not sure if it's possible within the same module). 

I didn't add those changes for wasm and native in this PR now, since I don't have such a change for jvm yet. I plan to figure it out in a separate branch.